### PR TITLE
UX-538 Fix controlled Listboxes

### PR DESCRIPTION
**Pull request by jonambas**
### PR Checklist

- [ ] Add the correct `type` label
- [ ] Pull request approval from #uxfe or #design-guild

### CODE REVIEW

File: libby/form/ListBox.lib.js [hunk 1]
```diff
@@ -163,4 +163,33 @@ describe('ListBox', () => {
...
+    function handleClick() {
+      if (value === 4) {
+        setValue(1);
+      } else {
+        setValue(value + 1);
+      }
+    }
+    return (
+      <>
+        <ListBox
```
Comment (logansparlin) [inline libby/form/ListBox.lib.js:179]: Should probably `setValue()` on change, values get out of syncwhen using the increment button and then choosing a different option by clicking

**Review by logansparlin — CHANGES_REQUESTED** [review]
Looks good! Just one small change to the example

File: libby/form/ListBox.lib.js (hunk 1 above)
Comment (jonambas) [reply libby/form/ListBox.lib.js:179]: Hmm this is probably a bug with the component – the ticket is to fix the listbox when only value is controlled

Comment (jonambas) [reply libby/form/ListBox.lib.js:179]: Oh nvm i see what youre saying

**Pull request merged**